notebooks/ch14/bias_risk.md

```diff
@@ -58,13 +58,13 @@ L(\hat{\theta}, X, y)
 \end{aligned}
 $$
 
-As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict.
+As before, we use $ \hat{\theta} $ as our vector of model parameters, $ x $ as a vector containing a row of a data matrix $ X $, and $ y $ as our vector of observed values to predict. $ X_i $is the $i$'th row of $ X $ and $ y_i $ is the $i$'th entry of y.
```

Comment:
double check space; We use $ X_i $ to represent the $i$'th row of $ X $ and $ y_i $ to represent the $i$'th entry of $y$.